homeassistant/components/zwave_js/const.py

```diff
@@ -33,3 +33,10 @@
 ATTR_PROPERTY_NAME = "property_name"
 ATTR_PROPERTY_KEY_NAME = "property_key_name"
 ATTR_PARAMETERS = "parameters"
+
+# service constants
+SERVICE_SET_CONFIG_VALUE = "set_config_value"
+
+ATTR_CONFIG_PROPERTY = "property"
+ATTR_CONFIG_PROPERTY_KEY_NAME = "property_key_name"
```

Comment:
```suggestion
ATTR_CONFIG_PROPERTY_KEY_NAME = "bitmask"
```
Not sure about this one, but this matches more closely to the zwave-js docs. Users aren't going to really understand what "property_key" and "property_key_name" are, unless they are looking a dump files.

Reply:
I would leave this out completely.
Just have each "bit mask property" have it's own config label and set a value for it.